### ISSUE TITLE
[Feature Request] VSelect options should have `role="option"`

### ISSUE THREAD

**Issue by binaryfunt** [vuetifyjs/vuetify#16881]
### Problem to solve
Children of the `role="listbox"` menu should have a role of `option` - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

Tested in Vuetify 3.1.7. (They used to have a role of `option` in Vuetify 2)

### Proposed solution
Add the `role="option"` attribute to the items in the menu
